irods/test/meta_test.py

```diff
@@ -520,13 +520,41 @@ def units():
                 if d: d.unlink(force = True)
                 helpers.remove_unused_metadata(session)
 
+    def test_AVUs_populated_improperly_with_empties_or_nonstrings_fail_identically__issue_547(self):
```

Comment:
Doesn't this test cover the same thing as the collection-based ones that follow?

Why loops instead of listing out each case like in the collection-based tests? The test feels like it could be simpler.

_I'm just wondering._

Reply:
Yes, as said, there is certainly overlap, and I'm wholly in favor of merging the tests.